pyanaconda/payload/__init__.py

```diff
@@ -1420,6 +1420,19 @@ def _setup_cdrom_device(self, storage, method, isodev, device):
                     method.server = host
                     method.dir = path
                 else:
+                    if flags.automatedInstall:
+                        # During a kickstart installation if we auto-detected
+                        # a local media, we should also disable the CDN in case
+                        # it was set previously and but not enough authetication
+                        # data is available.
+                        subscription_proxy = SUBSCRIPTION.get_proxy()
+                        org_set = bool(subscription_proxy.Organization)
+                        key_set = subscription_proxy.IsActivationKeySet
+                        kickstart_sufficient = org_set and key_set
+                        if not kickstart_sufficient:
+                            log.debug("payload: making sure cdrom is used instead of CDN")
+                            payload_proxy = PAYLOAD.get_proxy()
```

Comment:
payload_proxy is not used

Reply:
Oops, there should be one more line using the proxy - good catch! :) Fixed.